.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,6 +8,7 @@ assignees: ''
 ---
 
 <!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+If the matter is security related, please disclose it privately to a security contact: https://github.com/kudobuilder/kudo/blob/master/SECURITY_CONTACTS
```

Comment:
s/related/sensitiv ? but no strong preference. Would be great to have a mailing list once we have a domain...

Reply:
added `sensitive`